lib/excel_utils.rb

```diff
@@ -0,0 +1,9 @@
+module ExcelUtils
+  FORMULA_TRIGGERS = %w[- + = @].freeze
+
+  # Escapes potentially formula-triggering characters from the front of the
+  # string, i.e. +,-,=,@
+  def self.escape_formulas(string)
+    string&.gsub(/^[-+=@]/, '\\\\\0')
```

Comment:
Should this not be using the `FORMULA_TRIGGERS` constant? I also wonder whether we should do something like this:

```ruby
def self.escape_formulas(string)
   return string unless FORMULA_TRIGGERS.include?(string[0])
  
   '\' + string
end
```

To save running `gsub` against every single string and to be a bit clearer as to the intent of the code (I usually avoid interpolation like that, but it avoids a rabbit hole of slash escaping)

Reply:
👍🏽I've adjusted the method